lib/sycamore/sycamore/transforms/partition.py

```diff
@@ -381,7 +381,7 @@ class ArynPartitioner(Partitioner):
         extract_images: If true, crops each region identified as an image and attaches it to the associated
              ImageElement. This can later be fed into the SummarizeImages transform.
         local: If false, runs the partitioner remotely. Defaults to false
```

Comment:
probably need to update this comment

Reply:
Yes. Good catch - I'll fix this in a new pr.